storage/client/client.go

```diff
@@ -67,7 +67,7 @@ func (c *StorageClient) Shutdown() {
 
 // Status returns status information for the Oasis Indexer.
 func (c *StorageClient) Status(ctx context.Context) (*Status, error) {
-	qf := NewQueryFactory(strcase.ToSnake(c.chainID))
+	qf := NewQueryFactory(strcase.ToSnake(c.chainID), "" /* no runtime identifier for the consensus layer */)
```

Comment:
eh let's get other teammates' input on this

Reply:
because I wonder if there would ever be a use case for a single query factory object where a piece of code would want to query both a runtime's data and consensus data. it might come up when we're doing this stuff with getting what consensus height a runtime block was finalized

Reply:
filing #199